job_runner/runjob.py

```diff
@@ -113,7 +113,7 @@ def runjob():
     job_name = args.jobid
     storage_dir = resolve_path(cfg['storage']['root'])
     job_dir = storage_dir / job_name
-    job_dir.mkdir(exist_ok=True)
+    job_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
I would prefer to raise an error and ask the user to create the storage root specified in the configuration file. In general, I don't like creating many directories automatically without user consent.